contributors/devel/sig-release/cherry-picks.md

```diff
@@ -42,10 +42,8 @@ Again, the emphasis is on critical bug fixes, e.g.,
 - Panic, crash, hang
 - Security
 
-As a counter example, we would not support backporting a bug fix to an alpha
-feature in general. One condition where we would consider is if that bug caused
-an issue for in another part of the system even when the alpha feature was
-switched off / not in use.
+A bugfix for a functional issue (not a data loss or security issue) that only
+affects an alpha or beta feature does not qualify as a critical bug fix.
```

Comment:
I think this should be constrained to alpha features for now. The characterization of beta features is:

> The software is well tested. Enabling a feature is considered safe. Features are enabled by default.

I would expect backported bugfixes to beta features to be held to the same standard as any backport (fixes a severe issue, must not introduce risk to the rest of the system, etc), but I wouldn't entirely rule them out.

Reply:
:+1: